spec/classes/init_spec.rb

```diff
@@ -781,6 +781,38 @@
         end
       end
 
+      context "with manage_nsswitch parameter default value" do
+        let :facts do
+          { :osfamily => v[:osfamily],
+            :"#{v[:releasetype]}" => v[:release],
+            :lsbdistid => v[:lsbdistid],
+          }
+        end
+        it { is_expected.to contain_class('nsswitch') }
+      end
+      
+      context "with manage_nsswitch parameter set to true" do
```

Comment:
check for `'true'` and `true`. Here's an example, though you would drop the false elements from the array.

https://github.com/ghoneycutt/puppet-module-pam/blob/master/spec/classes/init_spec.rb#L784